source/MapPanel.cpp

```diff
@@ -1023,17 +1040,17 @@ void MapPanel::DrawEscorts()
 
 			// Active and parked ships are drawn/indicated by a ring in the center.
 			if(squad.second.activeShips || squad.second.parkedShips)
-				RingShader::Draw(pos, INNER - 1.f, 0.f, squad.second.activeShips ? active : parked);
+				RingShader::Draw(pos, OUTER * sqrt(zoom) * 2 + 2, OUTER * sqrt(zoom) * 2, squad.second.activeShips ? active : parked);
 
-			if(squad.second.outfits.size())
 				// Stored outfits are drawn/indicated by 8 short rays out of the system center.
+			if(squad.second.outfits.size())
```

Comment:
Please revert this change as its just moving the line (and the comment isnt at the right place or formatting because of it)